ui/src/pages/Upload.tsx

```diff
@@ -85,6 +114,20 @@ export const Upload: React.FC = () => {
     });
   };
 
+  const handleKeyDown = (event) => {
```

Comment:
let's type this event explicitly as a `KeyboardEvent<HTMLInputElement | HTMLTextAreaElement>` in the func arg so that we get nice TS autocomplete if some modifies this function in the future

Reply:
I think the` handlekeydown` event type from the autocomplete component is _not_ the same type as our input props. I thought it would be `React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>` or  `React.FormEvent<T>` or `React.SyntheticEvent<EventTarget>but` they don't seem to work. Can we keep it just as `any` ?

![image](https://user-images.githubusercontent.com/19617248/109392417-e9ca6280-78e9-11eb-9d00-33ac90e0ab52.png)

Reply:
We do this setter `params.inputProps.onKeyDown = handleKeyDown`, so we can say that the event type is the same as our input props and use `KeyboardEvent`